integration-tests/pom.xml

```diff
@@ -147,6 +147,7 @@
         <module>grpc-health</module>
         <module>google-cloud-functions-http</module>
         <module>google-cloud-functions</module>
+        <module>reactive-messaging-kafka</module>
```

Comment:
We'll have to add the IT job in the CI too, especially the native part. It can go here https://github.com/quarkusio/quarkus/blob/master/.github/workflows/ci-actions.yml#L476

Reply:
Added to the latest commit.